web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -2,8 +2,6 @@
 /* eslint-disable no-undef */
 
 import fs from "fs";
-import path from "path";
-import { fileURLToPath } from "url";
 import industryJson from "../../../../content/lib/industry.json" assert { type: "json" };
 import taskAgenciesJSON from "../../../../content/src/mappings/taskAgency.json" assert { type: "json" };
```

Comment:
[dust] Given the brittleness of the import of `industry.json` in `web/src/scripts/webflow/industrySync.mjs`, should these imports be updated as well?

(Also to get rid of that annoying warning that pops up when the script is launched about importing JSON modules?

Reply:
Given this file isn't the source of the issue and we plan to re-write these scripts soon, I don't feel too strongly about making the change. I think the import is cleaner than what I did in `industrySync.mjs` which essentially rebuilds this object.